blackjack.js

```diff
@@ -30,7 +73,13 @@ const calcPoints = (hand) => {
  * @returns {boolean} whether dealer should draw another card
  */
 const dealerShouldDraw = (dealerHand) => {
-  // CREATE FUNCTION HERE
+  let dealerTotal = function dealerTotalFunc (dealerHand){
```

Comment:
Hm this seems to be digging too deep to get the desired result. All you need from this is get the score object from `calcPoints(dealerHand)` and then see how it compares to the rules in the specification (<=16, >17, or 17 && isSoft). 

The function inside the function is not entirely necessary given the other code available.